precommit/mirsg-hooks.yaml

```diff
@@ -75,6 +75,7 @@ repos:
       - id: typos
         args:
           - --force-exclude
+          - --hidden
```

Comment:
should `--exclude .git/` be added so the `.git` folder isn't checked for typos? https://github.com/crate-ci/typos/issues/788

Reply:
That would only be an issue if running `typos` not through `pre-commit`